src/zcl_abapgit_code_inspector.clas.abap

```diff
@@ -32,6 +32,11 @@ CLASS zcl_abapgit_code_inspector DEFINITION
         !io_set TYPE REF TO cl_ci_objectset
       RAISING
         zcx_abapgit_exception .
+    METHODS check_obj_before_add
```

Comment:
suggest renaming to "skip_object" takes object as input, returns true if it is an object to be skipped

Reply:
I changed it.
In the new method, I create a CASE for possible future cases